web/src/components/molecules/Common/MultiValueField/index.tsx

```diff
@@ -29,7 +29,7 @@ const MultiValueField: React.FC<Props> = ({
     (e: ChangeEvent<HTMLInputElement>, id: number) => {
       onChange?.(
         value?.map((valueItem, index) =>
-          index === id ? (typeof e === "number" ? e : e.target.value) : valueItem,
+          index === id ? (typeof e === "number" ? e : e?.target.value) : valueItem,
```

Comment:
If e can be undefined, please update the prop type too to keep types consistent (right now it is `e: ChangeEvent<HTMLInputElement>`, but should be `e?: Change.....`)

Reply:
Done!